Products/CMFPlone/browser/configure.zcml

```diff
@@ -263,4 +263,11 @@
       permission="zope2.View"
       />
 
+  <browser:page
+      for="*"
+      name="icons"
```

Comment:
I would not name it that generic, but what it is:
```suggestion
      name="iconresolver"
```
(need changes below as well)

Reply:
During yesterday's discussion we said @@icons instead of @@icon would be acceptable. @@icons is similar to @@scales. It's short, it's simple. Let's keep it.

Reply:
In facts its defined once in main template, so I would prefer the adapter name to match the functionality. The resulting name in tal:define can be icons, that's what I understood yesterday.

Reply:
a good compromise... iconresolver says what it does, icons as usage makes it easy to use